tools/c7n_mailer/c7n_mailer/utils.py

```diff
@@ -9,10 +9,11 @@
 import yaml
 
 import jinja2
-import jmespath
 from dateutil import parser
 from dateutil.tz import gettz, tzutc
 
+from c7n.utils import jmespath_search
```

Comment:
technically mailer doesn't have a runtime dependency on c7n, so we should just leave the default here afaics.